docs/src/snap/howto/refresh-certs.md

```diff
@@ -0,0 +1,98 @@
+# Refreshing Kubernetes Certificates
+
+Keeping your {{product}} cluster secure and functional requires
+regularly refreshing its certificates. Certificates in Kubernetes ensure
+secure communication between the various components of the cluster. If
+these certificates expire, it can lead to communication failures, disrupted
+services, and potential security risks. This how-to will walk you through
+the steps to refresh the certificates for both control plane and worker
+nodes in your {{product}} cluster.
+
+## What you will need
+- A running {{product}} cluster
+
+```{note} To refresh the certificates in your cluster, make sure it was
+initially set up with self-signed certificates or with your own CA certificates
+and keys during the bootstrap process.
+```
+
+### Refreshing Control Plane Node Certificates
+To refresh the certificates on control plane nodes, follow these steps:
+
+1. Access each control plane node in your cluster.
```

Comment:
Maybe add the following on top instead:
`Perform the following steps on each control-plane node of the cluster.`